packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -4696,35 +4696,71 @@ class MetaData extends SingleChildRenderObjectWidget {
 /// * [SemanticsDebugger], an overlay to help visualize the semantics tree. Can
 ///   be enabled using [WidgetsApp.showSemanticsDebugger] or
 ///   [MaterialApp.showSemanticsDebugger].
+@immutable
 class Semantics extends SingleChildRenderObjectWidget {
   /// Creates a semantic annotation.
   ///
   /// The [container] argument must not be null.
-  const Semantics({
+  Semantics({
```

Comment:
Making this non-const is a breaking change.
We should leave this const.

Reply:
https://github.com/dart-lang/sdk/issues/20962 prevents me from keeping it `const`. An alternative is to duplicate all the properties, but risking `Semantics` and `SemanticsProperties` going out-of-sync. I also checked with all the code that I could find. There are no `const` instantiations of `Semantics` outside our own unit tests. I actually do not expect `const` instantiation to be very useful, because you at least need a label, which is unlikely to be `const`, or an action, which is even less unlikely to be `const`. So I thought removing `const` but still providing a `const Semantics.fromProperties` would be the right trade-off. Happy to change it to something else though.

Reply:
We should at a minimum announce it on flutter-dev then and get people's feedback, as per our breaking change policy.

Reply:
True. Sending an announcement shortly.